docs/guides/execution-modes.mdx

```diff
@@ -61,4 +61,11 @@ When a job is failed or canceled, the reported usage is as follows:
 
 * Job or batch mode: the reported usage is the time the QPU was locked for executing your workload until the time it failed or was canceled. Therefore, if the failure or cancellation occurred before the lock, the reported usage is zero. Otherwise, the workload's reported usage is the value that Qiskit Runtime returns as `consumed`. Thus, some failed jobs do not appear in your reported usage and others do.
 
-* Session mode: the reported usage is the wall-clock time from the when the first job started executing in the session until the session terminates, regardless of the number of jobs that fail or are cancelled.
+* Session mode: the reported usage is the wall-clock time from the when the first job started executing in the session until the session terminates, regardless of the number of jobs that fail or are canceled.
+
+<span id="ittl-by-mode"></span>
+## Interactive time to live (ITTL)
+
+The interactive time to live (ITTL) in batch mode is one second for all instances.
+
+The ITTL for a session on the Open Plan instance is one second. For Premium Plan users, the ITTL is 60 seconds, unless there is an instance-specific configuration that dictates a different ITTL.
```

Comment:
@jyu00 can you confirm that this ITTL info is correct? Also, in your opinion, is there anywhere else in the docs this info should go (either in addition to or instead of here)?